src/macros.js

```diff
@@ -845,6 +845,12 @@ defineMacro("\\rBrace", "\\html@mathml{" +
 defineMacro("\u2983", "\\lBrace"); // blackboard bold {
 defineMacro("\u2984", "\\rBrace"); // blackboard bold }
 
+// The stmaryrd function `\minuso` provides a "Plimsoll" symbol that
+// superimposes the characters \circ and mathminus. Used in chemistry.
+// TODO: A MathML `\minuso` that is not tightly bound to the KaTeX font.
+defineMacro("\\minuso", `{\\mathrlap{\\mathchoice{\\kern{0.145em}}{\\kern{0.145em}}
+{\\kern{0.1015em}}{\\kern{0.0725em}}\\circ}{-}}`);
```

Comment:
We could use the `\\html@mathml` command to define a unicode character, `\u29B5` in this case, for the MathML rendering.

Reply:
Good catch. I did not know that character existed.